src/app/layouts/carousel/carousel-styling-sample/carousel-styling-sample.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="carousel-container">
+<igx-carousel [loop] = "false">
+    <igx-slide>
+            <div class="slide-wrapper">
+        <h3>Ignite UI for Angular</h3>
+        <p>30+ Material-based Angular components to code speedy web apps faster.</p>
+        <p>As we continue to focus on helping design teams and developers deliver software faster with beautiful UX, we will ship high-productivity features like enhanced Ignite UI CLI features, integration with the Angular CLI, more Sketch UI kits and app templates for the designer-developer workflow, and visual configurators for charts and grids for any code editor, including Visual Studio Code.</p>
+            </div>
+    </igx-slide>
+    <igx-slide>
+            <div class="slide-wrapper">
+        <h3>Ignite UI for Javascript</h3>
+        <p>A complete JavaScript UI component library giving you the ability to build data-rich responsive web apps.</p>
+        <p>Get started on your next project with over 100 JavaScript UI components to help you create responsive, modern web apps supporting your favorite frameworks.</p>
+    </div>
+    </igx-slide>
+</igx-carousel>
+</div>
```

Comment:
Can we ensure proper formatting of the samples before we merge? It would be nice if all code samples on stackblitz are formatted properly.

Reply:
@simeonoff I updated the formatting.